source/common/http/header_map_impl.cc

```diff
@@ -150,6 +150,26 @@ void HeaderString::append(const char* data, uint32_t size) {
   string_length_ += size;
 }
 
+void HeaderString::rtrim() {
+  switch (type_) {
+  case Type::Reference: {
+    RELEASE_ASSERT("false", "rtrim does not support header string in reference type.");
```

Comment:
"false" is a c string, evaluates to true in a boolean context.

Reply:
:+1: 

fat finger